roxctl/central/initbundles/revoke.go

```diff
@@ -49,6 +69,40 @@ func applyRevokeInitBundles(ctx context.Context, cliEnvironment environment.Envi
 	return nil
 }
 
+func confirmImpactedClusterIds(impactedClusterNames, impactedClusterIds []string, out io.Writer, in io.Reader) (bool, error) {
+	if len(impactedClusterIds) != len(impactedClusterNames) {
+		return false, errors.New("number of impacted cluster IDs and names do not match")
+	}
+	if len(impactedClusterIds) == 0 {
+		return true, nil
+	}
+
+	_, _ = out.Write([]byte("Revoking init bundle(s) %s will impact the following cluster(s):\n"))
+
+	t := tabwriter.NewWriter(out, 4, 8, 2, '\t', 0)
+	_, _ = t.Write([]byte("Cluster ID\tCluster Name\n"))
+	for i := 0; i < len(impactedClusterIds); i++ {
+		_, _ = t.Write([]byte(impactedClusterIds[i] + "\t" + impactedClusterNames[i] + "\n"))
+	}
+	_ = t.Flush()
+
+	_, _ = out.Write([]byte("Are you sure you want to revoke the init bundle(s)? [y/N]"))
+
+	confirm, err := bufio.NewReader(in).ReadString('\n')
+	if err != nil {
+		return false, errors.Wrap(err, "reading user input")
+	}
+
+	confirm = strings.ToLower(confirm)
+	if confirm != "y\n" && confirm != "n\n" && confirm != "\n" {
+		return false, errors.New("invalid confirmation. Must be 'y' or 'n'")
+	}
+	if strings.ToLower(confirm) != "y\n" {
+		return false, nil
+	}
```

Comment:
There's already some [code dealing with reading a confirmation flag](https://github.com/stackrox/stackrox/blob/master/roxctl/common/flags/confirm.go#L35-L44), it'd be best to extract this to a helper and also use `strings.TrimSpace()` to trim the newlines to make things a bit more clear here.